api/task_processor/management/commands/runprocessor.py

```diff
@@ -22,6 +23,10 @@ class Command(BaseCommand):
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
 
+        # Used by the `register_recurring_task` decorator
+        # to decide wether it should register the task or not.
+        os.environ["RUN_BY_PROCESSOR"] = "True"
```

Comment:
We can move it to environment variables, but I am kinda using it as a replacement of https://stackoverflow.com/questions/24827748/django-significance-of-run-main-environment-variable , what do you think?

Reply:
Let's be a bit more explicit with this comment as to why it's necessary. Something along the lines of: 

"The task decorator runs in the global scope so will be executed whenever other e.g. manage.py commands are run and will try to save the task to the database which we want to avoid for non DB dependent commands."